### PR TITLE
Enable 'zopen' tools to be processed by help2man and create a script to generate man pages for zopen tools

### PR DESCRIPTION
**Pull request by MikeFultonDev**
This item provides the 'starter code' to:
- update the `zopen` and `zopen-*` tools to have basic `--help` and `--version` support. The actual help is rudimentary.
- provide a 'plumbing' utility `zopen-help2man` that generates man pages for each of the tools
- some general cleanup of the text for a few options.

@IgorTodorovskiIBM showed me how to clean up the commits... Looks right now

### CODE REVIEW

File: bin/lib/zopen-md2man [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    exit 0
+  else
+    echo "Unrecognized option $1. Processing terminated" >&2
+    printSyntax
+    exit 8
+  fi
+fi
+
+set -e
+zopen_version='0.6.3' ## add code to perform zopen --version once supported
```
Comment (IgorTodorovskiIBM) [inline bin/lib/zopen-md2man:27]: Should this be 0.7.0?

File: bin/lib/zopen-version [hunk 2]
```diff
@@ -1,4 +1,36 @@
...
+  echo "   or: $ME [OPTION]"
+  echo "Syntax: zopen-version [OPTIONS]"
+  echo "where OPTIONS can be:"
+  echo "  --help: print this help"
+  echo "  --version: print version"
+}
+
+tool='zopen' # Default name
+package="zopen tools"
+copyright="Copyright (C) 2023 zOS Open Tools Community."
```
Comment (IgorTodorovskiIBM) [inline bin/lib/zopen-version:17]:
```suggestion
copyright="Copyright (C) 2023 z/OS Open Tools Community."
```

File: bin/lib/zopen-md2man (hunk 1 above)
Comment (MikeFultonDev) [reply bin/lib/zopen-md2man:27]: Yeah. Hm. I changed it to 0.7.0 - maybe it got lost on the cleanup. Will fix

File: bin/lib/zopen-version (hunk 2 above)
Comment (MikeFultonDev) [reply bin/lib/zopen-version:17]: The slash was in there. I took it out because it confused help2man

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Pull request merged**